docs/integration.rst

```diff
@@ -22,29 +22,22 @@ Integration with GitHub Actions
 -------------------------------
 
 yamllint auto-detects when it's running inside of `GitHub
-Actions <https://github.com/features/actions>`_ and automatically uses the suited
-output format to decorate code with linting errors. You can also force the
-GitHub Actions output with ``yamllint --format github``.
+Actions <https://github.com/features/actions>`_ and automatically uses the
+suited output format to decorate code with linting errors. You can also force
+the GitHub Actions output with ``yamllint --format github``.
 
-An example workflow using GitHub Actions:
+An minimal example workflow using GitHub Actions:
 
 .. code:: yaml
 
    ---
-   name: yamllint test
-
-   on: push
+   on: push  # yamllint disable-line rule:truthy
```

Comment:
I thought that maybe including the disable comment would help reduce some of the never-ending duplicate issues about GitHub Actions and the `on` key

Reply:
:+1: